### PR TITLE
Added and fixed Polish strings

### PR DESCRIPTION
**Pull request by strebski**

### CODE REVIEW

**Comment by strebski** [comment]
CLA signed.

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -12603,9 +12658,11 @@
 
   [type.natural.tree_row]
     en = Tree row
+    pl - Rząd drzew
```
Comment (mpimenov) [inline data/strings/types_strings.txt]: s/-/=/

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -13143,6 +13207,7 @@
 
   [type.place]
     en = place
+    pl = miejsce
```
Comment (mpimenov) [inline data/strings/types_strings.txt]: s/miejsce/Miejsce/

File: data/strings/types_strings.txt [hunk 3]
```diff
@@ -2219,7 +2223,7 @@
     ja = 神社
     ko = 성지
     nb = Alter
-    pl = Kapliczka
+    pl = Kapliczka szintoistyczna
```
Comment (mpimenov) [inline data/strings/types_strings.txt]: Please remove the trailing space.

**Comment by mpimenov** [comment]
@strebski ping

File: data/strings/types_strings.txt (hunk 3 above)
Comment (strebski) [reply data/strings/types_strings.txt]: Yes, there is a unnecessary space, I'll fix it, thanks for info

Comment (strebski) [reply data/strings/types_strings.txt]: @mpimenov Have no idea how to edit my commit :-( Could you assist?

Comment (mpimenov) [reply data/strings/types_strings.txt]: It's ok if you make a separate commit with the changes.

Comment (strebski) [reply data/strings/types_strings.txt]: > It's ok if you make a separate commit with the changes.

OK, done - all fixes! And I've added some other localized strings. Already should be fine.

**Pull request merged**